woodchipper/configs.py

```diff
@@ -51,6 +67,7 @@ class JSONLogToStdout(BaseConfigClass):
         woodchipper.processors.GitVersionProcessor(),
         structlog.processors.TimeStamper(fmt="%Y-%m-%d %H:%M.%S", utc=False),
         structlog.processors.CallsiteParameterAdder(parameters=callsite_parameters),
+        SentryJsonProcessor(level=logging.ERROR, as_extra=True, active=not os.getenv("WOODCHIPPER_DISABLE_SENTRY")),
```

Comment:
Shouldn't this be the application's responsibility to configure this? This hardcodes a Sentry setup to the default `JSONLogToStdout` setup.

I do recognize that for Tackle's usage, Tackle would then have to maintain it's own configuration. That may not be worth it.

Reply:
Is your concern around the `as_extra=True`? Or around the inclusion of this processor at all?

Reply:
The inclusion of this processor at all. If an application uses a different logging/error integration than Sentry, then they have to write their own processor. An inclusion like Sentry to this library feels like we're hardcoding Tackle project quirks into woodchipper.

Reply:
Yeah I saw that and think it's slick. I do think it's fine to run with this: Tackle projects using Sentry are our only known users and the alternative is that those Tackle projects maintain their own configurations. I think that's fine in early <1.0 stages where the config design is likely to drift as we work out kinks.

Reply:
Agreed - thanks for the feedback and deliberation.